ts/node/sql.ts

```diff
@@ -620,11 +620,29 @@ function getAllConversations() {
     .prepare(`SELECT * FROM ${CONVERSATIONS_TABLE} ORDER BY id ASC;`)
     .all();
 
-  return (rows || []).map(m => {
-    const unreadCount = getUnreadCountByConversation(m.id) || 0;
-    const mentionedUsStillUnread = !!getFirstUnreadMessageWithMention(m.id);
-    return formatRowOfConversation(m, 'getAllConversations', unreadCount, mentionedUsStillUnread);
-  });
+  const formatted = compact(
+    (rows || []).map(m => {
+      const unreadCount = getUnreadCountByConversation(m.id) || 0;
+      const mentionedUsStillUnread = !!getFirstUnreadMessageWithMention(m.id);
+      return formatRowOfConversation(m, 'getAllConversations', unreadCount, mentionedUsStillUnread);
+    })
+  );
+
+  const invalidOnLoad =
+    formatted.filter(m => {
```

Comment:
Wont filter just return an empty array is no item pass the boolean check? do you still need `|| []` ?